src/pages/Swap/index.tsx

```diff
@@ -205,7 +218,7 @@ export default function Swap() {
   const outputValue = showWrap ? parsedAmount : trade?.outputAmount
   const fiatValueInput = useStablecoinValue(inputValue)
   const fiatValueOutput = useStablecoinValue(outputValue)
-  const priceImpact = useMemo(
+  const stablecoinPriceImpact = useMemo(
```

Comment:
why is this variable stableCoin specific now?

Reply:
so there are actually two different ways to calculate price impact - based on taking input token amount + output token amount and swapping to stablecoins, and comparing those diffs, or by using the swap's impact on the market midprice. i found it confusing that they are both referred to as priceImpact so i tried to differentiate by naming them differently in this file, though it's not an exhaustive rebrand